simulator/src/runs/alarm.rs

```diff
@@ -0,0 +1,12 @@
+use anyhow::{anyhow, Result};
+use supermusr_streaming_types::ecs_al00_alarm_generated::Severity;
+
+pub(crate) fn severity(severity: &str) -> Result<Severity> {
+    match severity {
```

Comment:
`clap` can parse enums directly, you should not have to do this.

Reply:
Severity is not an enum, but I can use custom enums as discussed earlier.